kf-protocol-derive/Cargo.toml

```diff
@@ -19,4 +19,4 @@ log = "0.4"
 
 [dependencies.syn]
 version = "1.0.38"
-features = ["full"]
+features = ["full", "extra-traits"]
```

Comment:
do we need all extra features? can we narrow down to smallest?

Reply:
`"extra-traits"` gives `Debug`, `Clone` on `syn` types.
If drop #[derive(Debug)] from some of the new structs, I can drop this feature.